qiling/os/posix/syscall.py

```diff
@@ -239,7 +239,17 @@ def ql_syscall_setuid(ql, null0, null1, null2, null3, null4, null5):
         UID = 1000
     ql.nprint("setuid(%i)" % UID)
     regreturn = UID
-    ql_definesyscall_return(ql, regreturn)     
+    ql_definesyscall_return(ql, regreturn)
+
+
+def ql_syscall_setpriority(ql, which, who, prio, null3, null4, null5):
```

Comment:
i am not certain about this, we might dont even to call os.piority. 

do you have a test case for this ?